lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -46,6 +46,24 @@ function dosomething_user_init() {
       dosomething_user_non_staff_redirect();
     }
   }
+
+  // If viewing a user/[northstar id]
+  if(arg(0) == 'user') {
```

Comment:
instead of putting this in an `init` hook, it probably makes more sense in the `hook_menu`